spec/component/models/duplicate_publication_group_spec.rb

```diff
@@ -1393,18 +1393,59 @@
       let!(:import1) { create :publication_import, publication: pub1 }
       let!(:import2) { create :publication_import, publication: pub2 }
 
-      context 'when PublicationMatchOnDoiPolicy returns true for these publications' do
+      context 'when both DOIs are present and equal' do
```

Comment:
For the next four contexts, the tests are good but I think the context descriptions might need to be modified.  I think we need to clarify that these have been merged because they passed one of the matching policies. In the last context, we should indicate that one of the publications did not pass either match policy so it did not merge.